server/views/components/list-header/list-header.njk

```diff
@@ -14,10 +14,14 @@
       </div>
     {% endif %}
 
-    <div class="grid">
-      <div class="grid__cell">
-        <p class="font-pewter {{ {s:'LR3', m:'LR2'} | fontClasses }} {{ {s:2, m:2, l:2} | spacingClasses({margin: ['top']}) }} {{ {s:2, m:2, l:2} | spacingClasses({margin: ['bottom']}) }}">{{ model.total }} items</p>
+    {% block extra %}{% endblock %}
```

Comment:
Hmmmm. Feels like a dangerous do-all component addition.

Reply:
Couldn't think of a better way to share the necessary styles out of `list-header` – I was imagining this as akin to the webcomponents `slot` spec: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot

Reply:
Components vs JSX?

Reply:
Could we call it slot then?